### ISSUE TITLE
Tabs don't appear correctly on the latest Firefox version

### ISSUE THREAD

**Issue by MiliAxe** [rockofox/firefox-minima#3]
After updating to the latest version of Firefox Developer Edition, tabs look as if they have no text.

![image](https://user-images.githubusercontent.com/55515322/202843683-26cf47bd-a26a-4cb9-8845-ad884abc1dba.png)
_I have switched to light mode so that it is easy to distinguish, clicking on the tabs work however._

 Thanks for the great userChrome.css!

**Comment by MiliAxe** [comment]
**Update**: pressing alt shows the tabs but incorrectly, sorry I couldn't capture a screenshot of it.

**Comment by MiliAxe** [comment]
**Update:** After tinkering with the code, I noticed that adding the `--tab-min-height` to `:root` fixed the problem but one other problem occured:

When I close tabs, my tabs seem to have added some sort of margin and they get disoriented: 
![out](https://user-images.githubusercontent.com/55515322/202848070-af9162b0-c143-403f-aff8-9338cea271ae.gif)
_(The GIF is sped up, the margins disappear after a few seconds of closing the tabs)_

I noticed that his happens with this CSS:
```css
.tabbrowser-tab[fadein]:not([pinned]) {
	max-width: var(--max-tab-width) !important;
}
```

**Comment by rockofox** [comment]
Hmm, I remember something similar happening when they added that little "view" button left of the tab bar.

I will get onto it asap

**Comment by MiliAxe** [comment]
It might be worth adding that when there are too many tabs added, with the addition of the "arrow" button (on the left in my screenshot), the tabs get disorganized again. It might be related to that?
![image](https://user-images.githubusercontent.com/55515322/202851014-e17f2255-3c80-41a9-b9da-496531f88909.png)

**Comment by rockofox** [comment]
I just made an update that should fix the issues with the disorientation (and polishes a few other things). 
- I removed the "arrow" button, as it breaks the layout and doesn't really fit my vision anyway.
- The issue with the margin that disappears after a few seconds is known. It is related to Firefox's animations. I personally have them disabled anyway, and that fixes the issue too (can be done by setting `toolkit.cosmeticAnimations.enabled` to `false` in about:config or by enabling "Reduce Motion" (or equivalent) in your operating system).

If any of those points bother you, feel free to open separate issues. I don't deem those as super important things, but I can take a look at it over long term if someone really cares enough ;)
